ontology/yaml/resources/HVAC/entity_types/CRAC.yaml

```diff
@@ -0,0 +1,39 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+########################
+### Canonical Types  ###
+########################
+
+CRAC_RHC_RTC:
+  description: "CRAC with return air temp/humidity control and discharge/cooling fan monitor."
+  is_canonical: true
```

Comment:
This is not a standard piece of equipment. This should not be canonical.

Reply:
Defined it as non standard